main.go

```diff
@@ -20,11 +23,15 @@ func main() {
 		gin.SetMode(gin.ReleaseMode)
 	}
 
-	if err := db.NewDBConnection(); err != nil {
+	var err error
+	model.DB, err = db.NewDBConnection()
```

Comment:
Therefore, even here the model is used, but actually its repository stuff, which was right before the changes